microprofile/WFLY-12313_MicroProfile_OpenAPI_1.1.adoc

```diff
@@ -0,0 +1,97 @@
+= Integrate MicroProfile OpenAPI 1.1
+:author:            Paul Ferraro
+:email:             paul.ferraro@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This enhancement proposes to add support for https://github.com/eclipse/microprofile-open-api/tree/1.1.2[Eclipse MicroProfile OpenAPI 1.1] to WildFly by integrating the implementation from https://github.com/smallrye/smallrye-open-api[SmallRye].
+This spec provides a unified Java API for producing documentation that complies with the https://github.com/OAI/OpenAPI-Specification/blob/3.0.2/versions/3.0.2.md[OpenAPI v3 specification].
+
+An MP OpenAPI subsystem exposes an HTTP GET `/openapi` endpoint that returns an OpenAPI v3 compliant document (in YAML or JSON format) containing documention of the RESTful services of an application.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-12313
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:fburzigo@redhat.com[Fabio Burzigotti]
+
+=== Testing By
+
+[ ] Engineering
+
+[X] QE
+
+=== Affected Projects or Components
+
+The SmallRye implementation of the MicroProfile Open API specification is already integrated by the following projects:
+
+* https://thorntail.io/[Thorntail] via the https://github.com/thorntail/thorntail/tree/master/fractions/microprofile/microprofile-openapi[microprofile-openapi fraction].
+* https://quarkus.io/[Quarkus] via the https://github.com/quarkusio/quarkus/tree/master/extensions/smallrye-openapi[smallrye-openapi extension].
+
+These integrations will heavily inform, though not dictate, the corresponding integration into WildFly.
```

Comment:
> it would be quite useful to mention how the projects listed here would be affected from the end user point of view

I did not imagine they would be affected.

> and also the chance for users to migrate their existing apps from TT to Wildfly.

Migration from TT should be trivial, as TT equivalent configuration won't touch the more ambiguous spec behavior mentioned below.

> Probably the community docs could also cover a typical migration scenario with just some high-level detail

We don't typically discuss migration in the community docs - but it'll need to be mentioned somewhere.  I'll find out the correct place for that stuff.